src/data/selectors/index.js

```diff
@@ -221,15 +220,11 @@ export const shouldShowSpinner = (state) => (
 
 /**
  * showBulkManagement(state, options)
- * Returns true iff the user has special access or bulk management is configured to be available
- * and the course has a masters track.
+ * Returns true iff the course has bulk management enabled
  * @param {object} state - redux state
  * @return {bool} - should show bulk management controls?
  */
-export const showBulkManagement = (state) => (
-  special.hasSpecialBulkManagementAccess(app.courseId(state))
-  || (tracks.stateHasMastersTrack(state) && state.config.bulkManagementAvailable)
-);
+export const showBulkManagement = (state) => (state.config.bulkManagementAvailable);
```

Comment:
Should we be using a simpleSelector?

Reply:
wait, this *is* a selector, never mind

Reply:
eeeh.  If there were more in the config module maybe.  SimpleSelectors are mostly helpful for when there are a number of key fields you want to expose on a model.  For a singleton, is more overhead than necessary.

Honestly, it may just make sense to wrap the config reducer in with the App reducer and then bundle this selector, but this is totally reasonable for now.